src/ruby/devcontainer-feature.json

```diff
@@ -1,21 +1,27 @@
 {
     "id": "ruby",
-    "version": "1.2.0",
-    "name": "Ruby (via rvm)",
+    "version": "1.2.0", // TODO - Is this a major or a minor update? Major, probably.
```

Comment:
Need feedback on versioning. I suspect a major change here, given the changes to the options.